docs/section12/Rule12-7.md

```diff
@@ -0,0 +1,38 @@
+
+# Receptacle - Rule 12-7
+
+**Rule ID:** 12-7  
+**Rule Description:** User RMR Receptacle Power in Baseline RMR?  
+**Rule Assertion:** Baseline RMR = User RMR  
+**Appendix G Section:** Receptacle  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR and U_RMR  
+**Applicability Checks:**  
+
+  1. Rule 12-4 = True  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- **Applicability Check 1:** ```if (rule-12-4.status == True):```
+- Get the receptacle load of each space in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
+  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
```

Comment:
use _user and _baseline suffixes on all variable

Reply:
Updated in #35